spec/ordersystem_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'ordersystem'
+
+describe OrderSystem do
+  let(:dish) { double :dish }
+  let(:option_one) { { dish: dish, price: 3 } }
+  let(:option_two) { { dish: dish, price: 5 } }
+  let(:takeaway) { TakeawayKitchen.new.menu }
```

Comment:
This is exactly where you should be using a double. This test is for the OrderSystem class, it should not need to use any other class. So here, you probably instead want something like:
```rb
  let(:short_menu) { [
      { dish: "Chicken Satay", price: 3 }, 
      { dish: "Spring Rolls", price: 2 }
  ]}
  let(:takeaway) { double :takeaway, menu: short_menu }
```

Or something similar.